### ISSUE TITLE
fairseq version 0.5.0 requirement

### ISSUE THREAD

**Issue by HaraldKorneliussen** [pytorch/translate#168]
It seems that on 12 jun there was a major update to fairseq, changing the library structure, and Translate has already started using this. The fairseq package installed in my miniconda environment following the installation instructions is 0.4.0. Running any translate command fails with 

```
from fairseq import (
ImportError: cannot import name 'tasks'

```

I think this old version of fairseq was installed as part of the combined PyTorch and Caffe2 Conda package. pip/conda don't know about the newer version.

Could you update the installation instructions so that the current version of Translate can be run?

**Comment by myleott** [comment]
After the attached PR lands, you should rerun `python setup.py install` and it will fetch the latest version.

**Comment by jmp84** [comment]
I actually tested installation from scratch and it seems to work. I'm not sure we want to pin to 0.5.
I assume what happened here was that you did a git pull on translate and didn't update fairseq?
@HaraldKorneliussen, does this work for you (once you're in the conda env): `pip uninstall -y pytorch-translate fairseq`, then `python3 setup.py build develop` ?

**Comment by HaraldKorneliussen** [comment]
I did a git pull on translate, yes. I did get it to work by building fairseq as well from source and installing that for development (with python3 setup.py build develop there). There was no fairseq-0.5.0 that conda could find. If I run 
`
pip uninstall -y pytorch-translate fairseq`

and 

`
python3 setup.py build develop
`

(in the translate directory), it installs fairseq 0.4.0:

```

Processing dependencies for pytorch-translate==0.1
Searching for fairseq==0.4.0
Best match: fairseq 0.4.0
Processing fairseq-0.4.0-py3.6-linux-x86_64.egg
Adding fairseq 0.4.0 to easy-install.pth file

```

I can't find fairseq-0.5.0 in egg form anywhere. I have to install it from source and building it for development (in the miniconda environment).
